explore/src/main/scala/explore/targeteditor/ElevationPlotSemester.scala

```diff
@@ -73,116 +73,111 @@ object ElevationPlotSemester:
-                )
-            }
-            .getOrElse(Resource.pure(fs2.Stream()))
+                          )
+                          .orEmpty
+                      ) ++
+                      fs2.Stream(xAxis.removePlotLine("progress"))
+                  )
+              }
+              .getOrElse(Resource.pure(fs2.Stream()))
+              .use(_.compile.drain)
       )
-      .useEffectWithDepsBy((props, _, _, chartOpt, _) => (props.date, chartOpt))(
-        (props, _, _, _, _) =>
-          (date, chartOpt) =>
-            CallbackTo(chartOpt.value.value) >>=
-              (_.map(chart =>
-                Callback {
-                  // 2pm of the selected day, same as for semester start and end
-                  val localDateTime: LocalDateTime =
-                    LocalDateTime.of(date, LocalTime.MIDNIGHT).plusHours(14)
-                  val zonedDateTime: ZonedDateTime =
-                    ZonedDateTime.of(localDateTime, props.site.timezone)
+      .useEffectWithDepsBy((props, _, _, chartOpt) => (props.date, chartOpt))((props, _, _, _) =>
+        (date, chartOpt) =>
+          CallbackTo(chartOpt.value.value) >>=
+            (_.map(chart =>
+              Callback {
+                // 2pm of the selected day, same as for semester start and end
+                val localDateTime: LocalDateTime =
+                  LocalDateTime.of(date, LocalTime.MIDNIGHT).plusHours(14)
+                val zonedDateTime: ZonedDateTime =
+                  ZonedDateTime.of(localDateTime, props.site.timezone)
 
-                  // Axes maybe undefined or empty when remounting.
-                  if (!js.isUndefined(chart.axes) && chart.axes.length > 0) {
-                    val xAxis = chart.xAxis(0)
-                    xAxis.removePlotLine("date")
-                    xAxis.addPlotLine(
-                      AxisPlotLinesOptions
-                        .XAxisPlotLinesOptions()
-                        .setId("date")
-                        .setValue(zonedDateTime.toInstant.toEpochMilli.toDouble)
-                        .setZIndex(1000)
-                        .setClassName("plot-plot-line-date")
-                    )
-                  }
+                // Axes maybe undefined or empty when remounting.
+                if (!js.isUndefined(chart.axes) && chart.axes.length > 0) {
+                  val xAxis = chart.xAxis(0)
+                  xAxis.removePlotLine("date")
+                  xAxis.addPlotLine(
+                    AxisPlotLinesOptions
+                      .XAxisPlotLinesOptions()
+                      .setId("date")
+                      .setValue(zonedDateTime.toInstant.toEpochMilli.toDouble)
+                      .setZIndex(1000)
+                      .setClassName("plot-plot-line-date")
+                  )
                 }
-              ).orEmpty)
-      )
-      .useEffectWithDepsBy((_, _, resize, _, _) => resize)((_, _, _, chartOpt, _) =>
```

Comment:
We don't need this resize here anymore? Does the chart handle resizing better?

Reply:
I should have commented this. This code was producing an exception when you try to resize. [
Now it won't but the graph won't be updated. I'll make a separate PR to fix it